tests/cases/unittests/tsserverProjectSystem.ts

```diff
@@ -204,7 +248,7 @@ namespace ts {
         }
 
         // TOOD: record and invoke callbacks to simulate timer events
-        readonly setTimeout = (callback: (...args: any[]) => void, ms: number, ...args: any[]): any => void 0;
+        readonly setTimeout = setTimeout;

```

Comment:
I would prefer not to use built-in `setTimeout` and instead trigger event manually similar to what we do for file system events.
- we are controlling both project system and host so there is no need to introduce unnecessary async bits in tests
- `setTimeout` will be triggered after the specified timeout which is great for real world usage but pointless for tests since we are introducing unnecessary delays.